src/imputation/tmi_imputation.py

```diff
@@ -395,9 +397,14 @@ def run_tmi(full_df, target_variables, sic_mapper):
 
     qa_df = qa_df.drop("trim_check", axis=1)
 
-    final_df = apply_tmi(df, target_variables, mean_dict)
+    final_tmi_df = apply_tmi(df, target_variables, mean_dict)
+
+    final_tmi_df.loc[qa_df.index, "211_trim"] = qa_df["211_trim"]
+    final_tmi_df.loc[qa_df.index, "305_trim"] = qa_df["305_trim"]
+
+    df = pd.concat([final_tmi_df, shortform_df])
 
-    final_df.loc[qa_df.index, "211_trim"] = qa_df["211_trim"]
-    final_df.loc[qa_df.index, "305_trim"] = qa_df["305_trim"]
+    final_df = df.sort_values(["reference", "instance"],
+                              ascending=[True, True]).reset_index(drop=True)
 
```

Comment:
@AnneONS Just wanted to check this approach is okay. In `run_tmi` we split `full_df` into two dataframes - one for short forms and one for long forms. We pass the `longform_df` through TMI and then concatenate the result of this and `shortform_df` back together on line 405.

I think previously we just outputted the longforms from imputation so wanted to double check that now outputting both short and long makes sense.